packages/@aws-cdk-testing/framework-integ/test/aws-stepfunctions-tasks/test/emrcontainers/integ.start-job-run.ts

```diff
@@ -22,7 +22,7 @@ const stack = new cdk.Stack(app, 'aws-stepfunctions-tasks-emr-containers-start-j
 stack.node.setContext(EC2_RESTRICT_DEFAULT_SECURITY_GROUP, false);
 
 const eksCluster = new eks.Cluster(stack, 'integration-test-eks-cluster', {
-  version: eks.KubernetesVersion.V1_22,
+  version: eks.KubernetesVersion.V1_29,
```

Comment:
`KubernetesVersion.V1_22` was causing the integ test to fail since version 1.22 is no longer supported. The integ test was failing with the following error message:

```
Unsupported Kubernetes version 1.22
```

I've update to the latest version.